ecms_base/modules/custom/ecms_workflow/ecms_workflow.module

```diff
@@ -22,6 +23,22 @@ function ecms_workflow_entity_bundle_create($entity_type_id, $bundle) : void {
     // Call the workflow service to update configuration.
     \Drupal::service('ecms_workflow.bundle_create')
       ->addContentTypeToWorkflow($bundle);
+
```

Comment:
Does this revoke the permissions related to the bundle every time an Emergency Notification is created? Or is it that it revokes the permissions from the specific node?

Reply:
It only revokes permissions when a bundle of that type is created, not for every instance of that node.